<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="RowEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>140</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetdimBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ColEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>140</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rows:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Columns:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowBtn">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>680</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>190</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Generate State</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InitBtn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>130</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial State</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SrchBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>680</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Route</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetTargtBtn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>330</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Target State</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Steps Number:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="StepTB">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>500</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>550</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search Time:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TimeTB">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>550</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>450</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matrix Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>600</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Speed:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpeedBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>600</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
